### PR TITLE
[Floatplane] Add extractor

### LINKED ISSUE #5877: Add support for Floatplane
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting a new site support request
- [X] I've verified that I'm running yt-dlp version **2022.11.11** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that none of provided URLs [violate any copyrights](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#is-the-website-primarily-used-for-piracy) or contain any [DRM](https://en.wikipedia.org/wiki/Digital_rights_management) to the best of my knowledge
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and am willing to share it if required

### Region

Europe/Netherlands

### Example URLs

https://www.floatplane.com/channel/GarbageTime
https://www.floatplane.com/channel/GarbageTime/posts
https://www.floatplane.com/post/WtrwiYxJPM

### Provide a description that is worded well enough to be understood

Some notes:
* channel\[channel-name\] redirects to /channel/\[channel-name\]/home
* The front-end is developed in React, with most of the UI being created from data fetched from an API. 
  I imagine this makes retrieving the actual media URL's somewhat challenging. 

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
[debug] Command-line config: ['-vU', 'https://www.floatplane.com/post/WtrwiYxJPM']
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version 2022.11.11 [8b64402] (pip)
[debug] Python 3.10.9 (CPython x86_64 64bit) - Linux-6.0.8-gentoo-x86_64-x86_64-AMD_Ryzen_7_3700X_8-Core_Processor-with-glibc2.36 (OpenSSL 1.1.1s  1 Nov 2022, glibc 2.36)
[debug] exe versions: ffmpeg 4.4.3 (fdk,setts), ffprobe 4.4.3, rtmpdump 2.4
[debug] Optional libraries: Crypto-3.16.0, brotli-1.0.9, certifi-3021.03.16, mutagen-1.46.0, secretstorage-3.3.3, sqlite3-2.6.0
[debug] Proxy map: {}
[debug] Loaded 1723 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Latest version: 2022.11.11, Current version: 2022.11.11
yt-dlp is up to date (2022.11.11)
[debug] [generic] Extracting URL: https://www.floatplane.com/post/WtrwiYxJPM
[generic] WtrwiYxJPM: Downloading webpage
WARNING: [generic] Falling back on generic information extractor
[generic] WtrwiYxJPM: Extracting information
[debug] Looking for embeds
ERROR: Unsupported URL: https://www.floatplane.com/post/WtrwiYxJPM
Traceback (most recent call last):
  File "/usr/lib/python3.10/site-packages/yt_dlp/YoutubeDL.py", line 1485, in wrapper
    return func(self, *args, **kwargs)
  File "/usr/lib/python3.10/site-packages/yt_dlp/YoutubeDL.py", line 1561, in __extract_info
    ie_result = ie.extract(url)
  File "/usr/lib/python3.10/site-packages/yt_dlp/extractor/common.py", line 674, in extract
    ie_result = self._real_extract(url)
  File "/usr/lib/python3.10/site-packages/yt_dlp/extractor/generic.py", line 2721, in _real_extract
    raise UnsupportedError(url)
yt_dlp.utils.UnsupportedError: Unsupported URL: https://www.floatplane.com/post/WtrwiYxJPM
```

### PR DESCRIPTION
**Pull request by JakeShirley**
**IMPORTANT**: PRs without the template will be CLOSED

### Description of your *pull request* and other information

</details>

This adds support (via a new extractor) for [Floatplane](https://www.floatplane.com) video posts.  It does _not_ add support to download/index entire channels.

Fixes #5877 

<details open><summary>Template</summary> 

### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [contributing guidelines](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#developer-instructions) including [yt-dlp coding conventions](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#yt-dlp-coding-conventions)
- [x] [Searched](https://github.com/yt-dlp/yt-dlp/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [x] Checked the code with [flake8](https://pypi.python.org/pypi/flake8) and [ran relevant tests](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#developer-instructions)

### In order to be accepted and merged into yt-dlp each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [ ] Fix or improvement to an extractor (Make sure to add/update tests)
- [x] New extractor ([Piracy websites will not be accepted](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#is-the-website-primarily-used-for-piracy))
- [ ] Core bug fix/improvement
- [ ] New feature (It is strongly [recommended to open an issue first](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#adding-new-feature-or-making-overarching-changes))

### CODE REVIEW

**Comment by JakeShirley** [comment]
I am a total Python script kiddy, so sorry if there are any obvious botches :) 

Also, I am not sure how to test this.  I didn't add login support (only cookie-based support) and I couldn't find any good docs on how to run tests with browser sessions/cookies.

Any help is appreciated!

**Comment by Grub4K** [comment]
To test extractors that require login information, create a file `test/local_parameters.json` and fill it with the params that you would want to test with. In this case use the `cookiesfrombrowser` parameter for loading cookies from the browser

File: yt_dlp/extractor/floatplane.py [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+        ###
+        # Fetch post details
+        ###
+        post_api_url = f"https://www.floatplane.com/api/v3/content/post?id={post_id}"
+
+        post_metadata = self._download_json(
+            post_api_url, post_id, headers=headers, note='Fetching post details')
+
+        if not traverse_obj(post_metadata, ('metadata', 'hasVideo')):
+            raise UnsupportedError(
```
Comment (pukkandan) [inline yt_dlp/extractor/floatplane.py:45]:
```suggestion
            raise ExtractorError(
```

File: yt_dlp/extractor/floatplane.py [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+        self._download_webpage(
+            'https://www.floatplane.com/', None,
+            note='Fetching session cookie')
+        self.session_id = self._get_cookies(
+            'https://www.floatplane.com')['sails.sid'].value
+
+    def _real_extract(self, url):
+        post_id = self._match_id(url)
+
+        headers = {'Cookie': f"sails.sid={self.session_id}"}
```
Comment (pukkandan) [inline yt_dlp/extractor/floatplane.py:34]: self._set_cookie

File: yt_dlp/extractor/floatplane.py [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+
+        ###
+        # Fetch post details
+        ###
```
Comment (pukkandan) [inline yt_dlp/extractor/floatplane.py:38]: unnecessary comment. Same below

File: yt_dlp/extractor/floatplane.py [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+        post_api_url = f"https://www.floatplane.com/api/v3/content/post?id={post_id}"
+
+        post_metadata = self._download_json(
+            post_api_url, post_id, headers=headers, note='Fetching post details')
```
Comment (pukkandan) [inline yt_dlp/extractor/floatplane.py:42]:
```suggestion
        post_metadata = self._download_json(
            f'https://www.floatplane.com/api/v3/content/post?id={post_id}',
            post_id, headers=headers, note='Fetching post details')
```
Singe use variables should be inlined. Same below

File: yt_dlp/extractor/floatplane.py [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+
+    def _real_extract(self, url):
+        post_id = self._match_id(url)
+
+        headers = {'Cookie': f"sails.sid={self.session_id}"}
+
+        ###
+        # Fetch post details
+        ###
+        post_api_url = f"https://www.floatplane.com/api/v3/content/post?id={post_id}"
```
Comment (pukkandan) [inline yt_dlp/extractor/floatplane.py:39]: Use single quotes everywhere

File: yt_dlp/extractor/floatplane.py [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+        remote_qualities_params = traverse_obj(
+            video_format_metadata, ('resource', 'data', 'qualityLevelParams'))
+        remote_cdn = video_format_metadata['cdn']
+        format_path_template = traverse_obj(
+            video_format_metadata, ('resource', 'uri'))
+        formats = []
+        for currentFormat in remote_qualities:
+            currentParams = remote_qualities_params[currentFormat['name']]
+            extension = currentParams["2"].split(
+                '.')[1]  # Get 'mp4' from '720p.mp4'
```
Comment (pukkandan) [inline yt_dlp/extractor/floatplane.py:77]: utils.determine_ext

File: yt_dlp/extractor/floatplane.py [hunk 7]
```diff
@@ -0,0 +1,137 @@
...
+        remote_qualities = traverse_obj(
+            video_format_metadata, ('resource', 'data', 'qualityLevels'))
+        remote_qualities_params = traverse_obj(
+            video_format_metadata, ('resource', 'data', 'qualityLevelParams'))
+        remote_cdn = video_format_metadata['cdn']
+        format_path_template = traverse_obj(
+            video_format_metadata, ('resource', 'uri'))
+        formats = []
+        for currentFormat in remote_qualities:
+            currentParams = remote_qualities_params[currentFormat['name']]
```
Comment (pukkandan) [inline yt_dlp/extractor/floatplane.py:75]: Use snake_case

File: yt_dlp/extractor/floatplane.py [hunk 8]
```diff
@@ -0,0 +1,137 @@
...
+            extension = currentParams["2"].split(
+                '.')[1]  # Get 'mp4' from '720p.mp4'
+
+            replaced_path = format_path_template.replace('{qualityLevelParams.2}', currentParams['2']).replace(
+                '{qualityLevelParams.4}', currentParams['4'])
+
+            formats.append({
+                'format_id': currentFormat['name'],
+                'width': currentFormat['width'],
+                'height': currentFormat['height'],
```
Comment (pukkandan) [inline yt_dlp/extractor/floatplane.py:85]: make non-fatal. Same below. Read https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#mandatory-and-optional-metafields

File: yt_dlp/extractor/floatplane.py [hunk 9]
```diff
@@ -0,0 +1,137 @@
...
+                'quality': 1,
```
Comment (pukkandan) [inline yt_dlp/extractor/floatplane.py:86]: ?

File: yt_dlp/extractor/floatplane.py [hunk 10]
```diff
@@ -0,0 +1,137 @@
...
+                'url': f"{remote_cdn}{replaced_path}",
+                'ext': extension
```
Comment (pukkandan) [inline yt_dlp/extractor/floatplane.py:88]: Core code tries to automatically set `ext` from URL. Does that not work here?

File: yt_dlp/extractor/floatplane.py [hunk 11]
```diff
@@ -0,0 +1,137 @@
...
+            'dislike_count': post_metadata['dislikes'],
+            'comment_count': post_metadata['comments'],
+            'release_timestamp': parse_iso8601(post_metadata['releaseDate']),
+            'release_date': unified_strdate(post_metadata['releaseDate']),
+
+            # I think all content on Floatplane is 'premium' only?
+            'availability': self._availability(needs_premium=True),
+
+            # Don't think Floatplane distinguishes between
+            # channels and uploaders, so fill out both?
```
Comment (pukkandan) [inline yt_dlp/extractor/floatplane.py:128]: It's your discretion. Most extractors only set one or the other depending on which is more appropriate

**Review by pukkandan — CHANGES_REQUESTED** [review]

File: yt_dlp/extractor/floatplane.py (hunk 9 above)
Comment (JakeShirley) [reply yt_dlp/extractor/floatplane.py:86]: Simple: I copy pasted and forgot to delete it!

File: yt_dlp/extractor/floatplane.py (hunk 10 above)
Comment (JakeShirley) [reply yt_dlp/extractor/floatplane.py:88]: It deduces `m3u8`, where as the API suggests the container to be `.mp4`.  Not sure what to do here!

File: yt_dlp/extractor/floatplane.py (hunk 5 above)
Comment (JakeShirley) [reply yt_dlp/extractor/floatplane.py:39]: I am a python noob here, what's the best way to use single quotes for this line? Cache the variable outside and use it? Seems like I can't index into a dict using single quotes inside a format string using single quotes:

```
video_format_api_url = f'https://www.floatplane.com/api/v2/cdn/delivery?type=vod&guid={video_metadata['guid']}'
```

Comment (pukkandan) [reply yt_dlp/extractor/floatplane.py:39]: Our convention is to use ' outside and " inside

https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#quotes

File: yt_dlp/extractor/floatplane.py (hunk 10 above)
Comment (JakeShirley) [reply yt_dlp/extractor/floatplane.py:88]: > To test extractors that require login information, create a file `test/local_parameters.json` and fill it with the params that you would want to test with. In this case use the `cookiesfrombrowser` parameter for loading cookies from the browser

Does this method allow me to run my extractor tests (ex: `python test/test_download.py TestDownload.test_Floatplane`).  I can't seem to get the test run to pick up my local config.

**Comment by JakeShirley** [comment]
> To test extractors that require login information, create a file `test/local_parameters.json` and fill it with the params that you would want to test with. In this case use the `cookiesfrombrowser` parameter for loading cookies from the browser

Does this method allow me to run my extractor tests (ex: `python test/test_download.py TestDownload.test_Floatplane`).  I can't seem to get the test run to pick up my local config.

**Comment by pukkandan** [comment]
> I can't seem to get the test run to pick up my local config.

What does your `local_parameters.json` look like?

**Comment by JakeShirley** [comment]
`test\local_parameters.json`

```json
{
   "cookiesfrombrowser ": "firefox"
}
```

Test command: `python test/test_download.py TestDownload.test_Floatplane`

**Comment by pukkandan** [comment]
Change to
```diff
-  "cookiesfrombrowser ": "firefox"
+  "cookiesfrombrowser": ["firefox"]
```

**Comment by JakeShirley** [comment]
Wow I can't believe how much a trailing space eluded me here :D

**Comment by JakeShirley** [comment]
@pukkandan Added a few tests!

File: yt_dlp/extractor/floatplane.py [hunk 12]
```diff
@@ -0,0 +1,155 @@
...
+class FloatplaneIE(InfoExtractor):
+    _VALID_URL = r'(?:https?://(?:www\.)?floatplane\.com/post/(?P<id>[a-zA-Z0-9-]+))'
+    _TESTS = [{
+        'url': 'https://www.floatplane.com/post/DdShecU983',
+        'info_dict': {
+               'id': 'DdShecU983',
+               'ext': 'mp4',
+               'channel': 'LinusTechTips',
+               'title': 'TQ: Are Intel Arc Graphics A Bad Idea?',
+               'description': '<p><br /></p>',
```
Comment (pukkandan) [inline yt_dlp/extractor/floatplane.py:21]: this is obv not the correct description...

File: yt_dlp/extractor/floatplane.py [hunk 13]
```diff
@@ -0,0 +1,155 @@
...
+        'skip': 'Requires an active Floatplane subscription to LinusTechTips',
+    },
+        {
+            'url': 'https://www.floatplane.com/post/g6IutvvaDF',
+            'info_dict': {
+                'id': 'g6IutvvaDF',
+                'ext': 'mp4',
+                'channel': 'LinusTechTips',
+                'title': 'The Most Average PC 2022',
+                'description': '<p>RIP the GTX 1060 6GB. The 1650 is now in first place, so we went digging through the rest of the Steam Hardware Survey to see if any other major changes have come about in the last two years. </p>',
```
Comment (pukkandan) [inline yt_dlp/extractor/floatplane.py:45]: same. Also, use md5 for long strings

File: yt_dlp/extractor/floatplane.py [hunk 14]
```diff
@@ -0,0 +1,155 @@
...
+            'skip': 'Requires an active Floatplane subscription to LinusTechTips',
+    }]
+
+    def _real_initialize(self):
+        self._download_webpage(
+            'https://www.floatplane.com/', None,
+            note='Fetching session cookie')
+
+        sessionIdCookie = self._get_cookies(
+            'https://www.floatplane.com').get('sails.sid')
```
Comment (pukkandan) [inline yt_dlp/extractor/floatplane.py:69]:
```suggestion
        self._download_webpage(
            'https://www.floatplane.com/', None, note='Fetching session cookie')

        sessionIdCookie = self._get_cookies('https://www.floatplane.com').get('sails.sid')
```

File: yt_dlp/extractor/floatplane.py [hunk 15]
```diff
@@ -0,0 +1,155 @@
...
+            },
+            'skip': 'Requires an active Floatplane subscription to LinusTechTips',
+    }]
+
+    def _real_initialize(self):
+        self._download_webpage(
+            'https://www.floatplane.com/', None,
+            note='Fetching session cookie')
+
+        sessionIdCookie = self._get_cookies(
```
Comment (pukkandan) [inline yt_dlp/extractor/floatplane.py:68]: snake_case everywhere

File: yt_dlp/extractor/floatplane.py [hunk 16]
```diff
@@ -0,0 +1,155 @@
...
+            'https://www.floatplane.com').get('sails.sid')
+
+        if sessionIdCookie is None:
+            raise ExtractorError(
+                'Cookie not found for floatplane.com.  Please login in your browser and retype', expected=True)
```
Comment (pukkandan) [inline yt_dlp/extractor/floatplane.py:73]: self.raise_login_required

File: yt_dlp/extractor/floatplane.py [hunk 17]
```diff
@@ -0,0 +1,155 @@
...
+
+        self.session_id = sessionIdCookie.value
```
Comment (pukkandan) [inline yt_dlp/extractor/floatplane.py:75]:
```suggestion
        self._session_id = sessionIdCookie.value
```

File: yt_dlp/extractor/floatplane.py [hunk 18]
```diff
@@ -0,0 +1,155 @@
...
+
+    def _real_extract(self, url):
+        post_id = self._match_id(url)
+
+        self._set_cookie("floatplane.com", "Cookie", f'sails.sid={self.session_id}')
```
Comment (pukkandan) [inline yt_dlp/extractor/floatplane.py:80]: * Wrong syntax
* Unnecessary anyway
```suggestion
```

File: yt_dlp/extractor/floatplane.py [hunk 19]
```diff
@@ -0,0 +1,155 @@
...
+                'comment_count': int,
+                'formats': list
+            },
+            'skip': 'Requires an active Floatplane subscription to LinusTechTips',
+    }]
+
+    def _real_initialize(self):
+        self._download_webpage(
+            'https://www.floatplane.com/', None,
+            note='Fetching session cookie')
```
Comment (pukkandan) [inline yt_dlp/extractor/floatplane.py:66]: u sure this is needed?

Comment (pukkandan) [inline yt_dlp/extractor/floatplane.py:66]:
```suggestion
        self._download_webpage(
            'https://www.floatplane.com/', None, note='Fetching session cookie')
```

File: yt_dlp/extractor/floatplane.py [hunk 20]
```diff
@@ -0,0 +1,155 @@
...
+                'Cookie not found for floatplane.com.  Please login in your browser and retype', expected=True)
+
+        self.session_id = sessionIdCookie.value
+
+    def _real_extract(self, url):
+        post_id = self._match_id(url)
+
+        self._set_cookie("floatplane.com", "Cookie", f'sails.sid={self.session_id}')
+
+        post_metadata = self._download_json(
```
Comment (pukkandan) [inline yt_dlp/extractor/floatplane.py:82]: line too long. read coding conventions

File: yt_dlp/extractor/floatplane.py [hunk 21]
```diff
@@ -0,0 +1,155 @@
...
+            raise ExtractorError(
+                'Provided Floatplane post does not contain a video', expected=True)
+
+        video_id = traverse_obj(post_metadata, ('videoAttachments', 0, 'id'))
+
+        video_metadata = self._download_json(
+            f'https://www.floatplane.com/api/v3/content/video?id={video_id}', post_id, note='Fetching video details')
+
+        video_format_metadata = self._download_json(
+            f'https://www.floatplane.com/api/v2/cdn/delivery?type=vod&guid={video_metadata["guid"]}', post_id, note='Fetching video format details')
```
Comment (pukkandan) [inline yt_dlp/extractor/floatplane.py:95]: same

File: yt_dlp/extractor/floatplane.py [hunk 22]
```diff
@@ -0,0 +1,155 @@
...
+        remote_cdn = video_format_metadata['cdn']
+        format_path_template = traverse_obj(
+            video_format_metadata, ('resource', 'uri'))
+        formats = []
+        for current_format in remote_qualities:
+            current_params = remote_qualities_params[current_format['name']]
+            extension = determine_ext(current_params["2"])  # Get 'mp4' from '720p.mp4'
+
+            replaced_path = format_path_template.replace('{qualityLevelParams.2}', current_params['2']).replace(
+                '{qualityLevelParams.4}', current_params['4'])
```
Comment (pukkandan) [inline yt_dlp/extractor/floatplane.py:111]: Dont set single use vars

File: yt_dlp/extractor/floatplane.py [hunk 23]
```diff
@@ -0,0 +1,155 @@
...
+
+            formats.append({
+                'format_id': current_format['name'],
```
Comment (pukkandan) [inline yt_dlp/extractor/floatplane.py:114]:
```suggestion
                'format_id': current_format.get('name'),
```

File: yt_dlp/extractor/floatplane.py [hunk 24]
```diff
@@ -0,0 +1,155 @@
...
+                'ext': extension
+            })
+
+        creator_id = traverse_obj(post_metadata, ('creator', 'id'))
+        creator_url_name = traverse_obj(post_metadata, ('creator', 'urlname'))
+
+        creator_api_url = f"https://www.floatplane.com/api/v2/plan/info?creatorId={creator_id}"
+
+        creator_metadata = self._download_json(
+            creator_api_url, post_id, note='Fetching creator details')
```
Comment (pukkandan) [inline yt_dlp/extractor/floatplane.py:127]: same

File: yt_dlp/extractor/floatplane.py [hunk 25]
```diff
@@ -0,0 +1,155 @@
...
+            # Post metadata
+            'like_count': post_metadata['likes'],
+            'dislike_count': post_metadata['dislikes'],
+            'comment_count': post_metadata['comments'],
+            'release_timestamp': parse_iso8601(post_metadata['releaseDate']),
+            'release_date': unified_strdate(post_metadata['releaseDate']),
+
+            # I think all content on Floatplane is 'premium' only?
+            'availability': self._availability(needs_premium=True),
+        }
```
Comment (pukkandan) [inline yt_dlp/extractor/floatplane.py:155]: * unnecessary comments and spaces
* Make everything non-fatal (Use format_field for channel_url, traverse_obj/.get for everything else)
* Use single quotes
* Dont set webpage_url unless you can explain a specific reason to do so

**Review by pukkandan — CHANGES_REQUESTED** [review]

File: yt_dlp/extractor/floatplane.py (hunk 12 above)
Comment (JakeShirley) [reply yt_dlp/extractor/floatplane.py:21]: I could use some guidance here.  The descriptions given are rich HTML blobs which can contain all sorts of HTML tags.  Any idea how other platforms handle this?

Comment (pukkandan) [reply yt_dlp/extractor/floatplane.py:21]: > Use `clean_html`

I assume u didn't see my edit

**Comment by scy** [comment]
@JakeShirley Do you have the time to work on these change requests? Would love to see Floatplane support in yt-dlp.

**Comment by Gelob** [comment]
@JakeShirley would you be interested in any help if you don't have the time?

**Comment by pukkandan** [comment]
Since the PR i stale, anyone could fork, address reviews and make a new PR

**Pull request closed without merging**